continuous_delivery/deploy_and_basics.rst

```diff
@@ -81,25 +73,22 @@ Deploy a Specific Git Tag
 **Build branch** allows you to specify to deploy an arbitrary Git branch or tag.
 
 .. important::
-   #. This should only be used with test systems [#f1]_.
-   #. The value of **Build branch** is ignored if `Docker Image ID <#run-menu-advanced>`__ is set.
+    This should only be used with test systems [#f1]_.
 
+Deploy a Specific Docker Image
+------------------------------
 
-Deploy a Specific Docker Image Tag
-----------------------------------
-
-This is only used for production systems, it allows you to deploy a docker image that was once installed on the test
-system. By default the ``test`` tag is deployed which installs the very same image the test system is using. You can
-however also install an older image formerly installed on the test system.
-
-#. Find the tag name for the version you want to have installed. It looks like this ``ecap-2017-05-09T12_54_27``.
-   **ecap** is the customer name.
-#. Enter the tag in the **Docker Image ID** field in the `run menu <#run-menu-advanced>`__.
-
-.. todo:: describe where the tag names can be found
-
+This should only be used for production systems, it allows you to deploy a docker image that was once installed on the test
+system [#f2]_. The image is determined by the DOCKER_PULL_URL parameter. It allows you to pick a image from a project an deploy it on your target project. 
+You can however also install an older image formerly installed on the test system.
 
 .. rubric:: Footnotes
 
 .. [#f1] In general, test systems are deployed from git and production systems reuse the Docker Images from
-         the test systems.
+         the test systems. But there might be some situation where you want to deploy a tag directly on production.
+         In that case remove the CD parameter "DOCKER_PULL_URL".
```

Comment:
That's rather important, I'd add a section that describes how to select and deploy a tag/branch.